### PR TITLE
errata modified for issue #1988, term inconsistency among ontology an…

### PR DESCRIPTION
**Pull request by mahdanoura**
addresses #1988 for providing the errata

### CODE REVIEW

**Review by danielpeintner — COMMENTED** [review]
~~Some general comments. I think the content of W3C erratas is most of the times split in the following sections:~~
* ~~Substantive Errata~~
* ~~Editorial Errata~~
* ~~Clarifications~~

~~see for example https://www.w3.org/XML/EXI/exi-10b-errata or https://www.w3.org/WAI/WCAG20/errata/~~ DONE

In general I think adding a date "when" the errata has been added makes sense also

File: errata11.html [hunk 1]
```diff
@@ -72,6 +72,9 @@ <h2>Substantive Issues</h2>
         </section>
         <section id="last">
           <h2>Editorial Issues</h2>
+          <ul>
+            <li>In the <a href="https://github.com/w3c/wot-thing-description/blob/main/ontology/td.ttl">Thing Description ontology</a> and it's associated <a href="https://www.w3.org/2019/wot/td#hasConfigurationInstance">HTML page</a>, the term hasConfigurationInstance needs to be changed to hasInstanceConfiguration to match the <a href="https://github.com/w3c/wot-thing-description/blob/main/context/td-context-1.1.jsonld">JSON-LD context file</a>. This change doesn not affect the <a href="https://www.w3.org/TR/wot-thing-description11/">WoT TD 1.1.</a> HTML specification</li>
```
Comment (egekorkan) [inline errata11.html:76]:
```suggestion
            <li> In the <a href="https://w3c.github.io/wot-resources/td/v1.1/ontology/td.ttl">Thing Description ontology</a> and it's associated <a href="https://www.w3.org/2019/wot/td#hasConfigurationInstance">HTML page</a>, the term hasConfigurationInstance has been changed to hasInstanceConfiguration to match the <a href="https://www.w3.org/2022/wot/td/v1.1">JSON-LD context file</a>. This change does not affect the <a href="https://www.w3.org/TR/wot-thing-description11/">WoT TD 1.1.</a> HTML specification.
            </li>
```

Comment (egekorkan) [reply errata11.html:76]: Generally, we should never link to the main branch of the TD repository.

**Comment by danielpeintner** [comment]
I noticed one minor issue in the published version.

> Latest Publication Date:
>    2 April 2020

https://w3c.github.io/wot-thing-description/errata11.html

Shall we quickly update the date as well? 

BTW, I am not sure whether the date is meant to relate to the TD publication  or the errata publication.
Anyhow, I think the date is wrong for both

**Pull request merged**